lessons/lesson-3/concepts.md

```diff
@@ -1,5 +1,19 @@
 # Concepts
 
+Hand interaction is a very natural way to interact with 3D models. As an interaction model, the use of hands is great for new users coupled with voice, eye tracking or head gaze. There is a low learning curve and consistent UX across hand tracking.
```

Comment:
I(the user) am not sure what is hand interaction and interaction model is. Please explain. 

>  There is a low learning curve and consistent UX across hand tracking.

Not sure what that means.